client/coral-configure/containers/ConfigureStreamContainer.js

```diff
@@ -49,23 +49,29 @@ class ConfigureStreamContainer extends Component {
           changed: false
         });
       }, 300);
+
+      // this.props.loadAsset(this.props.data.asset);
     }
   }
 
   handleChange (e) {
+
+    // TO DO: Don’t directly manipulate state and make state change immutable.
```

Comment:
The convention is to write `TO DO` together like `TODO`. Most IDE supports detecting todos out of the box if you follow this convention.